internal/app/command.go

```diff
@@ -25,6 +28,29 @@ func (c *command) String() string {
 	return c.Cmd + " " + strings.Join(c.Args, " ")
 }
 
+// runs exec command with retry
+func (c *command) retryExec(attempts int) exitStatus {
+	var result exitStatus
+
+	for i := 0; ; i++ {
```

Comment:
why not:
```go
for i := 0; i >= (attempts - 1); i++ {
// ...
}
```

Reply:
I kept things this way so I can use break before I print the Retrying message. Going with your suggestion I still need to add condition inside of the loop to check if that is going to be the last attempt so I don't print that message in the end

Reply:
Let me know if you're fine with what I've changed now @luisdavim